Tests/FluentAssertions.Shared.Specs/DateTimeAssertionSpecs.cs

```diff
@@ -1908,6 +1908,68 @@ public void When_asserting_subject_null_datetime_should_not_have_second_should_t
         }
         #endregion
 
+        #region Represent DateTimeKind
+        [TestMethod]
+        public void When_asserting_subject_datetime_represents_its_own_kind_it_should_succeed()
+        {
+            //-----------------------------------------------------------------------------------------------------------
+            // Arrange
+            //-----------------------------------------------------------------------------------------------------------
+            DateTime subject = new DateTime(2009, 12, 31, 23, 59, 00, DateTimeKind.Local);
+            DateTimeKind expectation = DateTimeKind.Local;
+
+            //-----------------------------------------------------------------------------------------------------------
+            // Act
+            //-----------------------------------------------------------------------------------------------------------
+            Action act = () => subject.Should().Represent(expectation);
```

Comment:
I was not sure what a proper name would be. The [MSDN documentation for DateTimeKind](https://msdn.microsoft.com/en-us/library/shx7s921(v=vs.110).aspx) says 

> Specifies whether a DateTime object represents a local time, a Coordinated Universal Time (UTC), or is not specified as either local time or UTC.

That's how I came up with the `Should().Represent` method. But I can live with your suggestion of `BeIn`.